packages/account/src/Components/form-sub-header/form-sub-header.jsx

```diff
@@ -1,10 +1,18 @@
+import classNames from 'classnames';
 import React from 'react';
 
-export const FormSubHeader = ({ title, subtitle }) => (
-    <div className='account-form__header'>
-        <h1 className='account-form__title'>
-            {title}
-            {subtitle && <i className='account-form__subtitle'>{subtitle}</i>}
-        </h1>
-    </div>
+export const FormSubHeader = ({ title, subtitle, description }) => (
+    <React.Fragment>
+        <div
+            className={classNames('account-form__header', {
+                'account-form__header--has-description': !!description,
+            })}
+        >
+            <h1 className='account-form__title'>
+                {title}
```

Comment:
Can we split the title and subtitle into `h1` and `h2` respectively?

Reply:
Of course, will do it right away. Thanks for the suggestion.